reorganize.py

```diff
@@ -0,0 +1,90 @@
+#reorganize.py
+#takes the output of the outbreak simulator and turns it into a csv file with relevant data shown in tabular format
+
+import numpy as np
+import pandas as pd
+import re
+import matplotlib.pyplot as plt
+import argparse
+
+def identify_timestamps(dictionary):
+    timestamps = [float(x.rsplit('_', 1)[1]) for x in dictionary.keys()]

```

Comment:
We know the keys have `_` but I would add `... if '_' in x` to avoid a potential exception raised by `[1]` if there is no `_` in the key.